dbt/adapters/default.py

```diff
@@ -89,6 +90,18 @@ def cancel_connection(cls, project, connection):
     # FUNCTIONS THAT SHOULD BE ABSTRACT
     ###
     @classmethod
+    def get_result_from_cursor(cls, cursor):
```

Comment:
nitpicky, but this probably doesn't belong under `FUNCTIONS THAT SHOULD BE ABSTRACT` right?